### PR TITLE
Update 02.main-text.md

### PR DESCRIPTION
**Pull request by jcolomb**
stating CAM is "simple" with a link to a huge documentation is close to lying... other changes are comments

### CODE REVIEW

**Comment by jcolomb** [comment]
@dhimmel : should we get the manubot part in a different -.md file ?

File: content/02.main-text.md [hunk 1]
```diff
@@ -20,10 +20,12 @@ The CRO is an open-source, community-developed ontology that currently contains
...
 The work done by the FORCE11 Attribution Working Group along with the OpenVIVO Task Force members at the time included reviewing existing scholarly contribution taxonomies and exploring ways to extend the CRediT taxonomy to create a prototype contributorship model that covers a wide selection of fields of research.
-The CRO can be used with the Contributor Attribution Model (CAM) - a simple, ontology-based specification for representing information about contributions made to research-related artifacts [@url:https://contributor-attribution-model.readthedocs.io/en/latest/].
+The CRO can be used with the Contributor Attribution Model (CAM) - an ontology-based specification for representing information about contributions made to research-related artifacts [@url:https://contributor-attribution-model.readthedocs.io/en/latest/].
 
 ### Incentives to use contributions
 
+[**JC: are these potential incentives, or does it really works this way, if the latter, is there data to back it up, if the former, this should be rephrased**]
+
```
Comment (nicolevasilevsky) [inline content/02.main-text.md:28]: I would say the former

File: content/02.main-text.md [hunk 2]
```diff
@@ -40,11 +42,13 @@ In other words, as long as institutions have not integrated accurate models of c
...
-A great advantage of using Manubot was that is was designed to traccck author contributions and made the contributions more transparent.
+A great advantage of using Manubot was that is was designed to track author contributions and made the contributions more transparent.
 [**Add something here about using CRO? Manubot should be be able to track our contributions in the yaml file too?**]
+[**JC: I am not sure that it actually works, github functionality is only counting commits number, not importance, and manubot would only track work done during manuscript writing. To get something automated we would need something much more complicated, where (for example) data would be published with a list of contributor, that list being imported into the manuscript while linking the manuscript to the data.**]
 Manubot trackers author metadata in a yaml file, which can be transfered to other manuscripts, saving time with collecting author details each time you write a manuscript.
```
Comment (nicolevasilevsky) [inline content/02.main-text.md:48]: See ticket: https://github.com/data2health/contributorship/issues/5

File: content/02.main-text.md [hunk 3]
```diff
@@ -40,11 +42,13 @@ In other words, as long as institutions have not integrated accurate models of c
...
-Manubot incorporates a simple workflow for tracking citations, and automatcally creates a reference list, removing the need to use a reference manager.
+[**JC: very little portability of the yaml actually, not even to other system using yaml to enter author information. One would need a codec to translate the list from one format to another, see github.com/open-science-promoters/contibutor_manager**]
+Manubot incorporates a simple workflow for tracking citations, and automatcally creates a reference list, removing the need to use a reference manager. [**putting citation just with doi has also its disadvantage, I do prefer a solution using a bib file (produced by a reference manager) and an author-date call to the bib file, otherwise one always need to click to see what the publication actually is**]
 Manubot is more technical to use compared to more commonly used authoring tools like Microsoft Word or Google Docs. All of the  authors were required to have a GitHub account and some basic familiarity with using GitHub, and with writing in Markdown.
```
Comment (nicolevasilevsky) [inline content/02.main-text.md:53]: noting that the comment above is from Julien. (Thanks for the comment. Do you want to incorporate this into the text?

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by jcolomb** [comment]
I would probably be better not to merge when the automatic checks are not passing. @dhimmel any clue about what is broken?

**Comment by dhimmel** [comment]
> any clue about what is broken?

hoping something temporary. I reran the CI for the merged commit https://github.com/data2health/contributorship/commit/573a22fd351ca1063d1bceb0b4158415293d2c96 and it worked.

**Pull request merged**